observability/opentelemetry/src/main/java/org/wildfly/extension/opentelemetry/OpenTelemetrySubsystemExtension.java

```diff
@@ -5,9 +5,6 @@
 
 package org.wildfly.extension.opentelemetry;
 
-import io.netty.util.internal.logging.InternalLoggerFactory;
-import io.netty.util.internal.logging.JdkLoggerFactory;
```

Comment:
IIRC, there's a netty dep in this module that was added just for this change, so that should probably be removed as it was in the clustering module above.

Reply:
+1 I do recall you adding this.